jira_sync/jira_wrapper.py

```diff
@@ -136,6 +137,7 @@ def create_issue(
             "description": url + "\n\n" + description,
             "issuetype": {"name": self.jira_config.default_issue_type},
             "labels": labels if labels else [],
+            self.jira_config.story_points_field: story_points,
         }
```

Comment:
Wouldn’t this require that the `story_points_field` is set, i.e. not work if left empty?

How about:
```suggestion
            "description": url + "\n\n" + description,
            "issuetype": {"name": self.jira_config.default_issue_type},
            "labels": labels if labels else [],
        }
        if self.jira_config.story_points_field:
            issue_dict[self.jira_config.story_points_field] = story_points
```

Reply:
I probably forgot to do this for create_issue, the logic is already in place for updating issue. Will do